Cargo.toml

```diff
@@ -88,6 +88,7 @@ http-body-util = "0.1"
 indexmap = "2"
 insta = { version = "1", features = ["filters"] }
 insta-cmd = "0.6"
+mime = "0.3"
```

Comment:
Is it worth it to add a new dependency just to use it for three `&str` constants?

Reply:
Removed!